extension/content_script.js

```diff
@@ -51,39 +51,25 @@ function getContainer(containerId) {
 
 // Each readable item
 // Assumes that readableDomId is already populated
+// TODO: Refactor this with setTracker().
 function setupClickListener() {
 	for (let i = 0; i < readableDomIds.length; i++) {
 		getContainer(i).click(function () {
 			containerId = i;
-			let container = getContainer(containerId);
```

Comment:
this is probably a bug. we are relying on global containers in other parts of the logic, and here I'm creating a local variable that will shadow the global variable.
(as we refactor along we might eventually get rid of the global variables... we'll see )

Reply:
LGTM